statsd/statsd.go

```diff
@@ -34,6 +35,19 @@ import (
 	"time"
 )
 
+/*
+MaxPayloadSize defines the maximum payload size for a UDP datagram, 1432 bytes
+is optimal for regular networks with an MTU of 1500 so datagrams don't get
+fragmented. It's generally recommended not to fragment UDP datagrams as losing
+a single fragment will cause the entire datagram to be lost.
+
+This can be increased if your network has a greater MTU or you don't mind UDP
+datagrams getting fragmented. The theoretical limit is 65467
+(65535 bytes Max UDP datagram size - 8byte UDP header - 60byte max IP headers)
+any number greater than that will see data loss.
+*/
+var MaxPayloadSize = 1432
```

Comment:
What functionality? Simply changing the variable should work out. But ok, i'll make it a constant.